polkadot-protocol/architecture/system-chains/coretime.md

```diff
@@ -1,4 +1,62 @@
 ---
 title: Coretime
-description: TODO
----
\ No newline at end of file
+description: Learn about the role of the Coretime system parachain, which facilitates the sale, purchase, assignment, and mechanisms of bulk coretime.
+---
+
+## Introduction
+
+The Coretime system chain facilitates the allocation, procurement, sale, and scheduling of bulk [coretime](../../glossary.md#coretime), enabling tasks (such as [parachains](../../glossary.md#parachain)) to utilize the computation and security that Polkadot provides. 
+
+The [Broker pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/index.html){target=_blank}, along with [Cross Consensus Messaging (XCM)](todo:addlink), enables this functionality to be delegated to the system chain, rather than on the relay chain. Using XCM's upwards message passing (UMP) to the relay chain allows for core assignments to take place for a task registered on the relay chain, utilizing the security that the relay chain provides.
```

Comment:
UMP is messaging protocol. XCM is a message format/language.

```suggestion
The [Broker pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/index.html){target=_blank}, along with [Cross Consensus Messaging (XCM)](todo:addlink), enables this functionality to be delegated to the system chain, rather than on the relay chain. Using XCMP's Upward Message Passing (UMP) to the relay chain allows for core assignments to take place for a task registered on the relay chain.
```